plugins/swell/package.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "@builder.io/ecom-swell-is",
```

Comment:
`@builder.io/plugin-swell`? even though their name is swell.is they just go by swell